modules/sdk/src/main/scala/org/tessellation/sdk/cli/http.scala

```diff
@@ -0,0 +1,33 @@
+package org.tessellation.sdk.cli
+
+import scala.concurrent.duration._
+
+import org.tessellation.ext.decline.decline._
+import org.tessellation.sdk.config.types.HttpClientConfig
+
+import com.comcast.ip4s.{Host, Port}
+import com.monovore.decline.Opts
+
+object http {
+  val externalIpOpts: Opts[Host] = Opts
+    .option[Host]("ip", help = "External IP (a.b.c.d.)")
```

Comment:
typo `a.b.c.d.` -> `a.b.c.d`

Reply:
good catch, it has been there since the very beginning